about.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>LOCUS: Technical Festival</title>
+  <link rel="icon" href="assets/icon.jpg" type="image/icon type">
+  <link rel="stylesheet" href="nav.css">
+  <link rel="stylesheet" href="about.css">
+  <link rel="stylesheet" href="footer.css">
+  <script src="https://kit.fontawesome.com/2649a194d9.js" crossorigin="anonymous"></script>
+
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <ul class="ul-toolbar">
+        <li><i class="fa-brands fa-facebook-f"></i></li>
+        <li><i class="fa-brands fa-twitter"></i></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+
+    <nav class="nav-bar">
+      <a class="header-logo" href="index.html"></a>
+      <a class="hamburger" href="#"><i id="menu-icon" class="fa-solid fa-bars" onclick="changeIcon(this)"></i></a>
+      <a class="menu-open">
+        <ul id="menu-list" class="menu-list">
+          <li><a id="list-item1" href="index.html">Home</a></li>
+          <li><a id="list-item2" href="#">Program</a></li>
+          <li><a id="list-item3" href="#">Sponsor</a></li>
+          <li><a id="list-item4" href="#">News</a></li>
+          <li><a id="list-item5" href="#">Live Score</a></li>
+          <li><a id="list-item6" href="#">Participate</a></li>
+        </ul>
+      </a>
+    </nav>
+  </header>
+
+  <section id="about-page" class="about-page">
+    <section class="about-main">
+      <div class="about-bg"></div>
+      <p class="greeting-about">Hello! Creative Minds</p>
+      <h1 class="headline-about">LOCUS: National Technical Exhibition 2023</h1>
+      <p class="short-description-about">LOCUS is the biggest Technical Fest in Nepal. It has a long history of 20 years. LOCUS is an umbrella organization led by undergraduate students of Electrical, Electronics, and Computer Engineering departments from IOE, Pulchowk Campus. All activities are guided by a single compass: fostering collaboration. Let's celebrate this March on the name of new innovation! </p>
+      <p class="contact">For more details about LOCUS 2023<br> Please feel free to contact on <br>
+        <strong><u>locus2023@pcampus.edu.np</u></strong>
+      </p>
+    </section>
+  
+    <section class="history">
+      <h2>Chronicles</h2>
+      <hr>
+      <p>Have your quick glance at the last two LOCUS events from 2022 and 2021!</p>
+      <div class="event">
+        <div class="event-one">
+          <h3 class="date">2022</h3>
+          <p>Featuring LOCUS 2022 - Year for Technology and Tnnovation</p>
+        </div>
```

Comment:
- [ ] [Optional] You missed small things here please see the about section and implement it. Also, the background you used in the about section is different please use the same color.
### Design you missed

![original](https://user-images.githubusercontent.com/81727988/218169489-117aa055-fbba-4cf8-aabb-2d361004339d.png)

Reply:
As you have suggested this change as an optional requirement, I will incorporate this section in the coming days.